models/article.js

```diff
@@ -14,4 +14,28 @@ exports.selectArticleById = (article_id) => {
     });
 };
 
+exports.fetchAllArticles = () => {
+  return db
+    .query(
+      `SELECT articles.*, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY created_at DESC;`
+    )
+    .then((result) => {
+      const articleArray = result.rows;
+      const newArticleArray = articleArray.map((articleObj) => {
+        delete articleObj.body;
+        articleObj.comment_count = +articleObj.comment_count
+        return articleObj;
+      });
+      return newArticleArray;
```

Comment:
You don't need to delete the body using JavaScript, you can be more specific in your SQL so that you only get the comments you want. I would encourage you to change it in this way 👍